packages/amazonq/src/lsp/activation.ts

```diff
@@ -4,9 +4,19 @@
  */
 
 import vscode from 'vscode'
+import path from 'path'
+import { AmazonQLSPDownloader } from './download'
 
 export async function activate(ctx: vscode.ExtensionContext): Promise<void> {
+    const serverPath = ctx.asAbsolutePath(path.join('resources', 'qdeveloperserver'))
+    const clientPath = ctx.asAbsolutePath(path.join('resources', 'qdeveloperclient'))
```

Comment:
string literals never need to be join()'d
```suggestion
    const serverPath = ctx.asAbsolutePath('resources/qdeveloperserver')
    const clientPath = ctx.asAbsolutePath('resources/qdeveloperclient')
```